infra.bs

```diff
@@ -280,6 +280,81 @@ quotes and monospace font.
 <a>ASCII lowercase</a> of <var>B</var>.
 <!-- TODO: define string equals? -->
 
+To <dfn data-export="">collect a sequence of characters</dfn> means that the following algorithm
```

Comment:
We should use just `export` in this document. I recognize you are porting from HTML, so sorry about the mismatch :)